.github/workflows/docker-build-publish.yml

```diff
@@ -14,7 +14,7 @@ jobs:
     permissions:
       contents: write
       packages: write
-    uses: rollkit/.github/.github/workflows/reusable_dockerfile_pipeline.yml@v0.4.1
+    uses: rollkit/.github/.github/workflows/reusable_dockerfile_pipeline.yml@develop
```

Comment:
TODO: need to bump this on .github and release new version of figure out how to reduce the image size.